polling_stations/apps/addressbase/management/commands/import_cleaned_addresses.py

```diff
@@ -0,0 +1,31 @@
+from uk_geo_utils.helpers import get_onsud_model
+from uk_geo_utils.management.commands import import_cleaned_addresses
+import os
+from django.db import connection
+
+
+class Command(import_cleaned_addresses.Command):
+    def import_addressbase(self):
```

Comment:
I understand the parent command doesn't do this, but given that this deletes a load of data, do we want to add a warning / confirmation to this command? Or does it not really matter?

Reply:
Maybe - it should be in uk-geo-utils too then. I don't think it really matters though, because it won't run unless you have a file to point it at, and if you have a file to point it at then I guess that's what you want. Adding some help text to the command that makes it clear that it's replacing, rather than adding, data would be probably be good.